src/installer/tests/Assets/Projects/StartupHookWithAssemblyResolver/StartupHookWithAssemblyResolver.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+  </PropertyGroup>
+
+  <!-- Copy the preloaded base type to the startup hook's output
+       directory to make it easy to find. -->
+  <ItemGroup>
+    <ProjectReference Include="../SharedLibrary/SharedLibrary.csproj" />
```

Comment:
nit: we normally use backslashes for path repo-wide (and MSBuild takes care of transformations).

Reply:
```suggestion
    <ProjectReference Include="..\SharedLibrary\SharedLibrary.csproj" />
```